src/components/app-nav-menu/NavMenu.tsx

```diff
@@ -167,15 +174,14 @@ export class NavMenu extends React.PureComponent<any, any> {
     const Logo = <Brand src={`${this.props.logoURL}`} alt='' />;
     const Header = (
       <PageHeader
+        className={isNavOpen ? 'nav-show' : 'nav-hide'}
         logo={Logo}
         logoProps={{ href: 'https://www.eclipse.org/che/', target: '_blank' }}
         headerTools={headerTools}
-        showNavToggle={true}
-        onNavToggle={this.onNavToggle}
       />
     );
 
-    const Sidebar = <PageSidebar nav={PageNav} theme={theme} />;
+    const Sidebar = <PageSidebar nav={PageNav} theme={theme} className={isNavOpen ? 'sidebar-show' : 'sidebar-hide'} />;
```

Comment:
why don't you use `isNavOpen` property to set the sitebar visibility?

Reply:
It doesn't work in this way after the upgrade. We can read this variable but cannot set it.

But I will be glad to see an alternative solution.